cluster-autoscaler/processors/nodegroupset/compare_nodegroups.go

```diff
@@ -40,6 +40,18 @@ const (
 	LabelTopologyGKE = "topology.gke.io/zone"
 	// LabelTopologyDiskCSIAzure is a CSI-specific label for disk driver in Azure.
 	LabelTopologyDiskCSIAzure = "topology.disk.csi.azure.com/zone"
+	// LabelAWSZoneID indicates the zone-id of the node in AWS
+	LabelAWSZoneID = "topology.k8s.aws/zone-id"
```

Comment:
```suggestion
	LabelTopologyAWSZoneID = "topology.k8s.aws/zone-id"
```
Consider changing this var (feel free to use something different if you have more ideas) for uniformity as other labels around topology seem to follow it